account_financial_report_webkit/migrations/8.0.1.3.0/post-migration.py

```diff
@@ -0,0 +1,75 @@
+# coding: utf-8
+import logging
+import time
+
+logger = logging.getLogger(__name__)
+
+# Query to recalculate static balances
+query = """
```

Comment:
Execution time of this query is 82 minutes

Reply:
Perfect.

Reply:
New execution time is also 80 seconds, but creates only 51822 records.

Reply:
Seconds or minutes?

Reply:
Minutes indeed

Reply:
Too bad ;-)